packages/sanity/src/core/create/start-in-create/StartInCreateAction.tsx

```diff
@@ -69,17 +70,18 @@ export function StartInCreateAction(
       : isDialogOpen && {
           type: 'dialog',
           onClose: closeDialog,
-          header: t('start-in-create-dialog.header'),
-          width: 'small',
-          content: (
-            <StartInCreateDialog
+          content: <StartInCreateActionContent />,
+          footer: (
+            <StartInCreateActionFooter
```

Comment:
Similarly, we separate document action content and footer to ensure it renders a little more closely to what you see in the studio elsewhere